addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -67,13 +67,17 @@ Bridge icloud:account:myaccount [appleId="mail@example.com", password="secure",
 }
 ```
 
+The device ID can be found in the Paper UI inbox.
+The information '@ "World"' is optional.
```

Comment:
*italic* would be better for highlighting the @ World

`*@ "World"*`

Reply:
changed to italic